README.md

```diff
@@ -78,13 +78,13 @@ sh get-docker.sh
 
 > You can read more about this at [`docker-install`](https://github.com/docker/docker-install)
 
-To compile the project, assuming that your already cloned this repository, just
+To compile the project, assuming that you already cloned this repository, just
 execute the `build` target, this will generate in `build` folder all the binaries
 and debian packages:
 
 ```sh
 > make build
-> ls -1 build/
```

Comment:
It's not necessary mistake ;) Often used in scripts to feed only filenames

Reply:
Yep, it's intentional, I wanted `-1`